courses/fundamentals_of_ada/labs/radar/adv_245_ravenscar_tasking/Makefile.zynq7000

```diff
@@ -0,0 +1,12 @@
+.PHONY: compile
```

Comment:
The filename seems weird. Is this a build artifact that got committed incorrectly?

Reply:
it is a makefile specifically for the zynq7000 which is the board I presented the training for, several such Makefile could be used, one per board, but this contains important information: namely how to call gnatemu and how to specify the board, adding a README to specify that.